source/util.js

```diff
@@ -68,3 +68,27 @@ exports.getTagVersionPrefix = pMemoize(async options => {
 		return 'v';
 	}
 });
+
+exports.getPreId = pMemoize(async options => {
```

Comment:
The method name could benefit from being more verbose. It's not clear what it means when reading it at the callsite.

Reply:
Updated method name to getPreReleasePostfix.